### PR TITLE
Update description in readme

### PR DESCRIPTION
**Pull request by bcdarius**
## Description
Update description in readme. It currently has description from [videojs-errors](https://github.com/brightcove/videojs-errors) plugin.

> A plugin that displays user-friendly messages when Video.js encounters an error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 [![NPM](https://nodei.co/npm/videojs-contrib-quality-levels.png?downloads=true&downloadRank=true)](https://nodei.co/npm/videojs-contrib-quality-levels/)
 
-A plugin that displays user-friendly messages when Video.js encounters an error.
+A library that exposes a list of quality levels available for the source.
```
Comment (gkatsev) [inline README.md]: This is also a plugin. This is probably fine, but I don't think it describes this plugin quite right.

Comment (bcdarius) [reply README.md]: How about?
> A plugin that provides a framework of working with source quality levels.

Comment (gkatsev) [reply README.md]: That sounds good.

**Comment by Miserlou** [comment]
This should be merged.

**Pull request merged**